.circleci/config.yml

```diff
@@ -97,6 +108,8 @@ workflows:
     jobs:
       - test-linux:
           context: org-global
+      - test-linux (long_running):
```

Comment:
I separated the two longest running tests so that CircleCI can run them parallel to the others, roughly cutting the clock time by 50 percent. I also added this `test-linux (long_running)` to the required checks in GitHub afterwards. Hope that's fine by you.

Reply:
@mbargull good idea. I ended up having to merge this into https://github.com/bioconda/bioconda-utils/pull/300. Github was expecting the long-running job but circleci didn't run it since it wasn't configured in .circleci/config.yaml yet. Result was a hung test.